recon_test_pack/run_test_listmode_recon.sh

```diff
@@ -0,0 +1,123 @@
+#! /bin/sh
+# A script to check to see if reconstruction of simulated data gives the expected result.
+#
+#  Copyright (C) 2011 - 2011-01-14, Hammersmith Imanet Ltd
+#  Copyright (C) 2011-07-01 - 2011, Kris Thielemans
+#  Copyright (C) 2014, University College London
+#  This file is part of STIR.
+#
+#  This file is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+#  This file is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  See STIR/LICENSE.txt for details
+#      
+# Author Nikos Efthimiou, Kris Thielemans
+#
+
+# Scripts should exit with error code when a test fails:
+if [ -n "$TRAVIS" ]; then
+    # The code runs inside Travis
+    set -e
+fi
+
+echo This script should work with STIR version ">"3.0. If you have
+echo a later version, you might have to update your test pack.
+echo Please check the web site.
+echo
+
+#
+# Options
+#
+MPIRUN=""
+
+#
+# Parse option arguments (--)
+# Note that the -- is required to suppress interpretation of $1 as options 
+# to expr
+#
+while test `expr -- "$1" : "--.*"` -gt 0
+do
+
+  if test "$1" = "--mpicmd"
+  then
+    MPIRUN="$2"
+    shift 1
+  elif test "$1" = "--help"
+  then
+    echo "Usage: `basename $0` [--mpicmd somecmd] [install_dir]"
+    echo "(where [] means that an argument is optional)"
+    echo "See README.txt for more info."
+    exit 1
+  else
+    echo Warning: Unknown option "$1"
+    echo rerun with --help for more info.
+    exit 1
+  fi
+
+  shift 1
+
+done 
+
+if [ $# -eq 1 ]; then
+  echo "Prepending $1 to your PATH for the duration of this script."
+  PATH=$1:$PATH
+fi
+
+# first delete any files remaining from a previous run
+rm -f my_*v my_*s my_*S
+
+echo "=== Simulate normalisation data"
+# For normalisation data we are going to use a cylinder in the center,
+# with water attenuation values
+echo "=== Gnerete fake emission image"
+generate_image  lm_generate_atten_cylinder.par
+echo "=== Calculate ACFs"
+calculate_attenuation_coefficients --ACF my_acfs.hs my_atten_image.hv Siemens_mMR_seg2.hs > my_create_acfs.log 2>&1
+if [ $? -ne 0 ]; then
+echo "ERROR running calculate_attenuation_coefficients. Check my_create_acfs.log"; exit 1;
+fi
+
+echo "=== Reconstruct listmode data"
+OSMAPOSL OSMAPOSL_test_lm.par > OSMAPOSL_test_lm.log 2>&1
```

Comment:
needs `${MPIRUN}` in front I believe (and all the others as well)

Reply:
:ok: